### DISCUSSION TITLE
Detect which calendar was selected on trigger

### DISCUSSION THREAD

**Discussion by SpiritSeal** [add2cal/add-to-calendar-button#533] [Q&A] [answered]
Is there a way that I can detect which calendar provider a user selects when they add an event to their calendar? I am building with the React wrapper. Thank you in advance for the help.

**Comment by jekuer** [comment, marked as answer]
Yes, you can.
Check the example and explanation at https://add-to-calendar-button.com/advanced-use#case-7
